### PR TITLE
Clean up: GRIDID

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- (re)moved schema from miscellaneous to new folder called digitalIdentifier for better grouping and decluttering of the miscellaneous folder

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison or @Peyman-N feel free to merge if you are also happy with the changes

File: schemas/digitalIdentifier/GRIDID.schema.tpl.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "_type": "https://openminds.ebrains.eu/core/GRIDID",
+  "required": [
+    "identifier"
+  ],
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "pattern": "^https:\/\/grid.ac\/institutes\/grid.[0-9]{1,}.([a-f0-9]{1,2})$",
+      "_instruction": "Enter the identifier for research organizations provided by the International Digital Object Identifier Foundation ('Gloabl Research Identifier Database IDentifier'; GRIDID) as an internationalized resource identifier (IRI) following the defined pattern (i.e., 'https://grid.ac/institutes/' + GRIDID)."
```
Comment (Peyman-N) [inline schemas/digitalIdentifier/GRIDID.schema.tpl.json:10]: Typo `Global`

**Review by Peyman-N — APPROVED** [review]

**Review by Peyman-N — CHANGES_REQUESTED** [review]
Typo `Global`

**Review by lzehl — APPROVED** [review]
I've corrected the typo. 

@Peyman-N or @apdavison or @aeidi89 feel free to merge if you also approve now

**Review by Peyman-N — APPROVED** [review]

**Review by aeidi89 — APPROVED** [review]
Looks good.

**Pull request merged**